src/mtev_cluster.c

```diff
@@ -40,6 +40,7 @@
 #include "mtev_cluster.h"
 #include "mtev_cht.h"
 #include "mtev_net_heartbeat.h"
+#include "mtev_arraylist.h"
```

Comment:
before I've switched to using mtev_hash I've used these arraylists to store the payloads, remember? I just forgot to remove this line, sorry!